Demo/Sources/ListsView.swift

```diff
@@ -9,6 +9,9 @@ import SwiftUI
 
 // Behavior: h-exp, v-exp
 struct ListsView: View {
+    @AppStorage(UserDefaults.serviceUrlEnvironmentKey)
```

Comment:
```suggestion
    @AppStorage(UserDefaults.serviceUrlKey)
```

Environment key could otherwise be confused with SwiftUI `@Environment`.